gammapy/modeling/models/time.py

```diff
@@ -123,6 +125,74 @@ def evaluate_norm_at_phase(self, phase):
         fp = self.table["NORM"]
         return np.interp(x=phase, xp=xp, fp=fp, period=1)
 
+    #    @property
+    def ontime(self, t_min, t_max):
```

Comment:
I think there is no real need to have this in a separate method anymore please just move the code to `sample_time()` and remove the separate method.

Reply:
Done